lightly/utils/lars.py

```diff
@@ -125,7 +122,6 @@ def step(self, closure: Optional[Callable[[], float]] = None) -> Optional[float]
             with torch.enable_grad():
                 loss = closure()
 
-        # exclude scaling for params with 0 weight decay
```

Comment:
I would add it back and just format it correctly:
`# Exclude scaling for params with 0 weight decay.`